tests/notifier/notifiers/inventory_summary_test.py

```diff
@@ -114,164 +121,176 @@ def test_upload_to_gcs_with_csv(self, mock_os, mock_tempfile, mock_storage):
         mock_tmp_csv.name = fake_tmpname
         mock_tmp_csv.write = mock.MagicMock()
 
+        # act
         notifier._upload_to_gcs([{}])
 
+        # assert
         mock_tmp_csv.write.assert_called()
         mock_storage.return_value.put_text_file.assert_called_once_with(
             fake_tmpname, gcs_path)
+        mock_progess_queue.put.assert_called()
```

Comment:
Updated a few existing tests to ensure the progress queue is called. The rest is formatting/readability changes for these tests to align with the Arrange/Act/Assert testing pattern.

Reply:
Yes